docs/source/1.0/spec/core/idl.rst

```diff
@@ -705,6 +709,84 @@ The following example defines an ``integer`` shape with a :ref:`range-trait`:
         }
 
 
+.. _idl-enums:
+
+Enum shapes
+-----------
+
+The :ref:`enum` shape is defined using an :token:`smithy:enum_shape_statement`.
+
+The following example defines an :ref:`enum` shape:
+
+.. code-block:: smithy
+
+    $version: "2.0"
+
+    namespace smithy.example
+
+    enum Suit {
+        DIAMOND
+        CLUB
+        HEART
+        SPADE
+    }
+
+The following defines an :ref:`enum` shape with traits:
+
+.. code-block:: smithy
+
+    $version: "2.0"
+
+    namespace smithy.example
+
+    enum Suit {
+        @enumValue("DIAMOND")
+        DIAMOND
+
+        @enumValue("CLUB")
+        CLUB
+
+        @enumValue("HEART")
+        HEART
+
+        @enumValue("SPADE")
+        SPADE
+    }
+
+
+IntEnum shapes
```

Comment:
nit: Should probably have its own heading, meaning the above would be the singular `idl-enum`.

Reply:
This is its own heading, it's just not `ref`able. I'll add the ref target anyway